smoothscroll.js

```diff
@@ -27,9 +27,12 @@
    * @returns {Date}
    */
   function now() {
-    return w.performance !== undefined && w.performance.now !== undefined ?
-      // if performance object supported return now, if not fallback to date object
-      w.performance.now() : Date.now !== undefined ? Date.now() : new Date().getTime();
+    // if performance object supported return now, if not fallback to date object
+    if (w.performance !== undefined && w.performance.now !== undefined) {
+      return w.performance.now();
+    }
+
+    return Date.now();
```

Comment:
removed ```Date.getTime``` statement because we're already using ```window.getComputedStyles``` which is not supported in **IE8**, so it makes no sense.